src/index.js

```diff
@@ -12,12 +12,21 @@ function insert(rule) {
 function createRule(className, decls, media) {
   var newDecls = []
   for (var property in decls) {
-    newDecls.push(hyphenate(property) + ":" + decls[property] + ";")
+    typeof decls[property] !== "object" &&
+      newDecls.push(hyphenate(property) + ":" + decls[property] + ";")
   }
   var rule = "." + className + "{" + newDecls.join("") + "}"
   return media ? media + "{" + rule + "}" : rule
 }
 
+function concat(str1, str2) {
```

Comment:
@maxholman Can you make this a single statement? It's just a matter of style.

Reply:
Like this?

```
function concat(str1, str2) {
  return str1 + (/^[a-z]/.test(str2) ? " " : "") + str2
}
```

Tried this one previously. It's an extra 2 bytes post min+gzip. Is that cool?

Reply:
It's fine, I think the issue is probably the position of the functions in the file. This is a never-ending rabbit hole. 😄

Reply:
Done. I also made the regex smaller which offsets those extra bytes ;)